src/stores/TransactionStore.js

```diff
@@ -117,9 +117,15 @@ export default class TransactionStore {
         createEventFuncTypes,
         eventParams,
       ).substr(2);
+
       const data = `0x${CREATE_EVENT_FUNC_SIG}${paramsHex}`;
       // Send tx
-      const txid = await promisify(nbotMethods.transfer['address,uint256,bytes'].sendTransaction, [eventFactoryAddr, escrowAmt, data, { gas }]);
+      const txid = await promisify(nbotMethods.transfer['address,uint256,bytes'].sendTransaction, [eventFactoryAddr, escrowAmt, data, {
```

Comment:
add validation before calling this function. if we dont have a valid exchanger or exchangeRate then we should show an error message.